www/starctl.php

```diff
@@ -51,6 +51,7 @@ function setStarCtlValue(id, val)
         // with automatic mouse rollover highlighting.
 
         ?>
+<span class=fmtnotes>To turn on accessible controls, please visit the <a href="https://www.ifdb.org/editprofile">settings page</a>.</span>
```

Comment:
the href should be just `/editprofile`.

Reply:
Thanks! I fixed the href. I also got a chance to log in and test it, and added <br> so that the stars will start on a new line when this message shows up on the "Write a Review" page.